templates/email/oxfordshire/problem-confirm.html

```diff
@@ -0,0 +1,37 @@
+[%
```

Comment:
Presumably this email template can now be dropped, given the user will be getting the subsequent email with the information contained in it?